src/components/Dropdown/Dropdown.tsx

```diff
@@ -16,8 +16,7 @@ export interface IDropdownState {
 export class Dropdown extends React.Component<IDropdownProps, any> {
```

Comment:
@dzearing I've changed to extending BaseComponent but i get a warning that _events is private in dropdown, but is not private in the BaseComponent.

Suggestion?